File: FrontEnd/Frontend_C#/SoftPac/SoftPacPersistance/DAO/DetallesPropuestaDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface DetallesPropuestaDAO
    {
        int Insertar(DetallesPropuestaDTO detallePropuesta);
        int Modificar(DetallesPropuestaDTO detallePropuesta);
        int eliminarPorPropuesta(int propuestaId);
        int eliminarLogico(DetallesPropuestaDTO detallePropuesta);
        DetallesPropuestaDTO ObtenerPorId(int detallePropuestaId);
    }
}

File: Backend_Java/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/DetallesPropuestaDAO.java
package pe.edu.pucp.softpac.dao;

import pe.edu.pucp.softpac.model.DetallesPropuestaDTO;

public interface DetallesPropuestaDAO {
    public Integer insertar(DetallesPropuestaDTO detallePropuesta);

    public Integer modificar(DetallesPropuestaDTO detallePropuesta);

    public Integer eliminarPorPropuesta(Integer propuestaId);

    public Integer eliminarLogico(DetallesPropuestaDTO detallePropuesta);
}
